### ISSUE TITLE
Window buttons cover title of the application on Mac OS

### ISSUE THREAD

**Issue by DylanBulmer** [DylanBulmer/TheDocs#6]
## Problem:
The buttons of the application window on Mac OS X is covering the title

## Solution:
Change the style of the frame type to default

**Comment by EricEisaman** [comment]
Here is my main.js for an Electron app with a transparent window border that works fine on Mac:

const electron = require('electron')
const {app,BrowserWindow,Menu} = electron

let win,win2

app.on('ready', () => {
  win = new BrowserWindow({
    width: 800,
    minWidth: 800,
    height: 600,
    title: '',
    titleBarStyle: 'hidden',
    defaultFontFamily: 'fantasy',
    transparent: true,
  })
  const menuTemplate = [
    {
        label: 'MAIN',
        submenu: [
            {
                label: 'About ...',
                click: () => {
                    if(win2)return
                    win2 = new BrowserWindow({
                      width: 256,
                      height: 256,
                      title: '',
                      titleBarStyle: 'hidden',
                      defaultFontFamily: 'fantasy',
                      transparent: true,
                      alwaysOnTop: true,
                    });
                    win2.loadURL(`file://${__dirname}/about.html`)
                    win2.on('closed', () => {
                     win2 = null
                   })
                    console.log('Made by SirFizX');
                }
            }, {
                type: 'separator'
            }, {
                label: 'Quit',
                click: () => {
                    app.quit();
                }
            }
        ]
    }
 ];
  const menu = Menu.buildFromTemplate(menuTemplate)
  Menu.setApplicationMenu(menu)
  win.loadURL(`file://${__dirname}/index.html`)
  //win.openDevTools()
})

exports.openWindow = () => {
    let win = new BrowserWindow({height: 600, width: 300})
    win.loadURL(`file://${__dirname}/otherpage.html`)
}

**Comment by EricEisaman** [comment]
![electron-app](https://user-images.githubusercontent.com/6116865/37246847-124b9818-247f-11e8-9d4c-528509c26295.png)

**Comment by DylanBulmer** [comment]
So I did some more researching into making my own custom title bar and to do it, I have to actually create my own. So I'm going to keep what I have and I decided to remove the frame as well so Windows gets the same effect. I may use the following too just to make every button custom to my application:

```
new BrowserWindow({titleBarStyle: 'customButtonsOnHover', frame: false})
```

Right now I'm in the process of developing the title bar within the application with HTML. 

Here is how I found this out:
* https://stackoverflow.com/questions/35660043/how-to-customize-the-window-title-bar-of-an-electron-app
* https://github.com/electron/electron/blob/master/docs/api/frameless-window.md

And yes, I will be documenting this with TheDocs!
